src/tools/crossgen2/ILCompiler.ReadyToRun/ObjectWriter/SectionBuilder.cs

```diff
@@ -211,6 +211,11 @@ public class SectionBuilder
         /// </summary>
         TargetDetails _target;
 
+        /// <summary>
+        /// Generate stable, unique names for various type system entities in the graph.
+        /// </summary>
+        NameMangler _nameMangler;
```

Comment:
Nit - readonly. If you could fix the same omission I made for _target a few lines above, that would be even better :-).